### PR TITLE
Include claim review publish date on claim review schema

### LINKED ISSUE #660: Include claim review publish date on claim review schema
### Background information
The claim review schema JsonLd setup in the review page is missing the publish date of the review.

### PR DESCRIPTION
**Pull request by pepermao**
- add date on review schema
- change the claim review publish date

closes #660

### CODE REVIEW

File: server/claim-review/schemas/claim-review.schema.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export class ClaimReview {
     @Prop({
         type: mongoose.Types.ObjectId,
         required: true,
-        ref: "ClaimRevision",
+        ref: "Claim",
```
Comment (thesocialdev) [inline server/claim-review/schemas/claim-review.schema.ts:16]: Shouldn't the ClaimReview reference the Sentences now?

File: src/pages/claim-review.tsx [hunk 2]
```diff
@@ -55,7 +55,7 @@ const ClaimPage: NextPage<{
                 jobTitle: personality.description,
                 image: personality.image,
             },
-            datePublished: claim.date,
+            datePublished: sentence.date,
```
Comment (thesocialdev) [inline src/pages/claim-review.tsx:58]: Are you sure the sentence date is the same as the claim review date? Also, are they the same as the publish date in the state machine?

**Review by thesocialdev — CHANGES_REQUESTED** [review]

File: src/pages/claim-review.tsx (hunk 2 above)
Comment (pepermao) [reply src/pages/claim-review.tsx:58]: The sentence date has the claim review date
```js
return Promise.all([
    this.claimReviewService.getReviewStatsBySentenceHash({
        sentence_hash,
        isDeleted: false,
        isPublished: true,
    }),
    this.claimReviewService.getReviewBySentenceHash(sentence_hash),
    this.claimReviewService.getUserReviewBySentenceHash(
        sentence_hash
    ),
]).then(([stats, claimReviewObj, userReview]) => {
    return {
        userReview,
        date: claimReviewObj.date,
        personality: claimReviewObj.personality,
        stats,
        ...sentence.toObject(),
    };
});
```
## Claim Review Published date
image

## claim Review Task published date
image

File: server/claim-review/schemas/claim-review.schema.ts (hunk 1 above)
Comment (pepermao) [reply server/claim-review/schemas/claim-review.schema.ts:16]: The claim Review schema has the report field with the id, and the report schema has the sentence_hash field. Is a claim Review schema reference sentence necessary?

File: src/pages/claim-review.tsx (hunk 2 above)
Comment (pepermao) [reply src/pages/claim-review.tsx:58]: miliseconds diference

File: server/claim-review/schemas/claim-review.schema.ts (hunk 1 above)
Comment (thesocialdev) [reply server/claim-review/schemas/claim-review.schema.ts:16]: @pepermao you're right. At least not at the moment, maybe when we start looking for performance gains we can revisit it.

**Review by thesocialdev — APPROVED** [review]

**Pull request merged**